nipype/utils/filemanip.py

```diff
@@ -29,6 +29,12 @@
 fmlogger = logging.getLogger("filemanip")
 
 
+related_filetype_sets = [
```

Comment:
Even DRYer--assuming all extensions start with a period, no need to specify the period here!